tools/cloud-build/hpc-toolkit-pr-validation.yaml

```diff
@@ -53,6 +53,20 @@ steps:
     time addlicense -check -ignore **/lysozyme-example/submit.sh . \
       || { echo "addlicense failed"; exit 1; }
     time make tests
+- id: ofe-virtual-env
+  waitFor:
+  - git-fetch-unshallow
+  name: >-
+    us-central1-docker.pkg.dev/$PROJECT_ID/hpc-toolkit-repo/hpc-toolkit-builder
+  entrypoint: /bin/bash
+  args:
+  - '-c'
+  - |
+    set -e
+    python3 -m venv /opt/ofe
+    source /opt/ofe/bin/activate
+    pip install --upgrade pip
+    pip install --dry-run --no-cache-dir -r community/front-end/ofe/requirements.txt
```

Comment:
IIUC this will crash without a new version of pip. Is that new version of pip implemented in our weekly builder? If not I don't think it is a good idea to add this test knowing it is not compatible with the automated builder.

Reply:
It is in our weekly builder #1056 and the copy inside the virtual environment is upgraded again in the line above your comment. Doing both may be overkill, of course.

Reply:
I manually upgrade the weekly image and you can see the results in the PR check ("ofe-virtual-env")